packages/GisidaLite/src/constants.ts

```diff
@@ -0,0 +1,15 @@
+export const DEFAULT_ACCESS_TOKEN = 'pk.eyJ1Ijoib25hIiwiYSI6IlVYbkdyclkifQ.0Bz-QOOXZZK01dq4MuMImQ';
```

Comment:
Do you think that we should have this as a constant in a public repo?

Reply:
The access token used here is public. For example purposes, I think it would be fine https://docs.mapbox.com/mapbox-gl-js/example/simple-map/